client/signup/site-mockup/style.scss

```diff
@@ -1,7 +1,7 @@
 // Mockup wrap
 .site-mockup__wrap {
 	margin: auto;
-	padding: 0 16px;
+	padding: 16px;
```

Comment:
By way of explanation, now that we're ditching `<Card />` around our steps, we need this gutter to compensate for the lost bottom margin.